.dockerignore

```diff
@@ -1,8 +1,29 @@
-__pycache__
+*/__pycache__
```

Comment:
Does this cover __pycache__ in the project root too?

Reply:
I would think so.

Reply:
Damn, no. I need **.